sql/parser/builtins.go

```diff
@@ -575,3 +610,13 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+func abs(x DInt) (DInt, error) {
+	switch {
+	case x == math.MinInt64:
+		return x, errAbsOfMinInt64
```

Comment:
just thinking out loud: we can probably support this if we introduce `DUint`. Not sure it's worth it.

We should at least give a better error message, which doesn't mention a go primitive type verbatim.